### ISSUE TITLE
Wrong returns of API:get_special_tokens_mask()

### ISSUE THREAD

**Issue by ishikawa-takumi** [huggingface/transformers#4274]
# 🐛 Bug

## Information

Model I am using (Bert, XLNet ...): Roberta

Language I am using the model on (English, Chinese ...): English

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [x ] my own modified scripts: (give details below)
I use transformers tokenizer, and created mask using API: get_special_tokens_mask.
[My Code](https://github.com/ishikawa-takumi/transformers-sample/blob/master/tokenizer.ipynb)

In [RoBERTa Doc](https://huggingface.co/transformers/model_doc/roberta.html), returns of this API is "A list of integers in the range [0, 1]: 0 for a special token, 1 for a sequence token". But I seem that this API returns "0 for a sequence token, 1 for a special token".
## To reproduce

1. Use get_special_tokens_mask().
2.
3.

## Expected behavior

Returns of this API is "A list of integers in the range [0, 1]: 0 for a special token, 1 for a sequence token".

## Environment info

     
- `transformers` version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

**Comment by Evpok** [comment]
Yes, this is https://github.com/huggingface/transformers/issues/1784 come again, the issue is in the doc, not in the code

**Comment by LysandreJik** [comment]
Indeed, thanks for raising this issue.
